crypto/rand/rand_lcl.h

```diff
@@ -106,8 +114,36 @@ struct rand_drbg_st {
     size_t min_entropylen, max_entropylen;
     size_t min_noncelen, max_noncelen;
     size_t max_perslen, max_adinlen;
-    unsigned int reseed_counter;
+
+    /*
+     * Normal reseeding
+     *
+     * A normal reseeding of the DRBG occurs whenever the number of
+     * generate requests since the last reseeding exceeds a certain
+     * threshold, the so called reseed_interval.
+     */
+
+    /* counts the number of generate requests since the last reseed */
+    unsigned int generate_counter;
+    /* maximum number of generate requests until a reseed is required */
     unsigned int reseed_interval;
+
+    /*
+     * Automatic reseeding
+     *
+     * Automatic reseeding needs to be enabled explicitely. In addition
+     * to normal reseeding, a drbg will reseed if a given time interval
+     * has elapsed or if its parent has been reseeded in the meantime.
+     * For details, see the comment before RAND_DRBG_enable_auto_reseed().
+     */
+
+    /* Counts the number of seconds since the last reseed */
```

Comment:
Stores the time of the last reseed?

Reply:
Ups, I missed this one. A stupid copy and paste error. Will be fixed in a moment.